autode/atoms.py

```diff
@@ -129,6 +129,7 @@ def translate(self, vec: np.ndarray):
 
          Arguments:
              vec (np.ndarray): Shape = (3,)
+
```

Comment:
not sure abot this one 😉

Reply:
I was trying to make Sphinx render the arguments correctly 😂